index.html

```diff
@@ -21,6 +21,8 @@
 
         <!-- Add your site or application content here -->
         <p>Hello world! This is HTML5 Boilerplate.</p>
+        <img src="http://placekitten.com.s3.amazonaws.com/homepage-samples/200/140.jpg" width="400" height="280" alt="kitten"/>
+        <img src="img/cute_puppy.jpg" width="680" height="510" alt="puppy"/>
```

Comment:
Is this a Creative Commons Puppy? Check search.creativecommons.org for appropriately licensed puppies.